charts/rancher-logging/rancher-logging/3.9.401+up3.9.4/values.yaml

```diff
@@ -115,6 +115,16 @@ additionalLoggingSources:
     enabled: false
   gke:
     enabled: false
+  kubeAudit:
+    enabled: false
+    tolerations:
+    - key: node-role.kubernetes.io/controlplane
+      value: "true"
+      effect: NoSchedule
+    - key: node-role.kubernetes.io/etcd
+      value: "true"
+      effect: NoExecute
```

Comment:
Conflicted on whether or not `logDirectory` should be here as a blank value. We may want to change its name to be something with "PathPrefix" to match our style too.

CC: @paynejacob for thoughts too